contrib/mpd/files/sysusers.conf

```diff
@@ -0,0 +1,5 @@
+# Create MPD system user
+
+u _mpd - "MPD user" /var/lib/mpd /usr/bin/nologin
```

Comment:
i think you need a tmpfiles entry for this dir too

Reply:
also, does it make /var/lib/mpd/music|playlists itself on start or do you need tmpfiles for that too?

Reply:
i haven't specified a music directory in the config initially, but I might add that as a default and put them in tmpfiles